lib/Sema/DerivedConformanceDistributedActor.cpp

```diff
@@ -31,5 +31,5 @@ bool DerivedConformance::canDeriveDistributedActor(
 // ==== ------------------------------------------------------------------------
 
 ValueDecl *DerivedConformance::deriveDistributedActor(ValueDecl *requirement) {
- return nullptr;
+  return nullptr;
```

Comment:
(this will be used more very soon; it's tricky to syntheside dist actor because the inits need the synthesized fields etc.. keeping for now)